R/redcap_dates.R

```diff
@@ -0,0 +1,80 @@
+#' REDCap Date Conversion
+#'
+#' This function is trying to take all (most)
+#' possibilities to enter dates into account and converts them to as.Date
+#' format.
+#'
+#' @param var variable to convert
+#'
+#' @return converted variable
+#' @export
+#' @importFrom crayon bold underline red
+
+#'
+#' @examples
+#' var <- "dataframe$var"
+#' redcap_dates(var)
+
+
+redcap_dates <- function(var) {
```

Comment:
generally, I wonder if there's something that already does this in `lubridate`... functions `ymd` or `dmy`...

Reply:
i think you always need to specify somehow in the lubridate how the input looks like that you would like to convert. the function is trying to take all possible ways (at the least the ones I came across so far) of input into account and makes the conversion... but i'm open for suggestions